### PR TITLE
update black-screen to 0.1.23

### PR DESCRIPTION
**Pull request by nyaaao**
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/black-screen.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 cask 'black-screen' do
-  version '0.1.6'
-  sha256 '47852369cb5e8477d87a1f72817d9f799b22216e3ff53c1a906173d43d2dcb3a'
+  version '0.1.23'
+  sha256 '76ef3c805131201287a69c9486422f854bf0bebd4ace95c71244ede10668462e'
 
-  url "https://github.com/shockone/black-screen/releases/download/v#{version}/black-screen-#{version}-mac.zip"
+  url "https://github.com/shockone/black-screen/releases/download/v0.1.24/black-screen-#{version}-mac.zip"
```
Comment (vitorgalvao) [inline Casks/black-screen.rb]: Leave this as was.

File: Casks/black-screen.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 cask 'black-screen' do
-  version '0.1.6'
-  sha256 '47852369cb5e8477d87a1f72817d9f799b22216e3ff53c1a906173d43d2dcb3a'
+  version '0.1.23'
```
Comment (vitorgalvao) [inline Casks/black-screen.rb]: Set this as `0.1.24`.

File: Casks/black-screen.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/black-screen.rb]: Ah, I see the problem, the last one is still `0.1.23`.

**Comment by vitorgalvao** [comment]
Thank you.

**Comment by nyaaao** [comment]
Sorry for the late reply, thanks for merging this quickly

**Pull request merged**